engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetRenderBufferMask.java

```diff
@@ -37,6 +40,15 @@
     private SetRenderBufferMask defaultInstance;
     private SetRenderBufferMaskTask task;
 
+    /**
+     * Creates an instance of this class with the given parameters.
+     *
+     * @param fboName A ResourceUrn identifying the FBO whose render masks have to be modified.
+     * @param fboManager The FBOManager responsible for managing the given FBO.
+     * @param renderToColorBuffer A boolean indicating whether the Color buffer of the given FBO should be written to.
+     * @param renderToDepthBuffer A boolean indicating whether the DepthStencil buffer of the given FBO should be written to.
+     * @param renderToLightBuffer A boolean indicating whether the Light buffer of the given FBO should be written to.
+     */
```

Comment:
You might want to revisit this javadoc too in light of the previous comment: this class is actually very specialized.